spring-boot-project/spring-boot-docs/src/main/asciidoc/index.adoc

```diff
@@ -52,7 +52,7 @@ Phillip Webb; Dave Syer; Josh Long; Stéphane Nicoll; Rob Winch; Andy Wilkinson;
 :propdeps-plugin: https://github.com/spring-projects/gradle-plugins/tree/master/propdeps-plugin
 :ant-manual: http://ant.apache.org/manual
 :code-examples: ../java/org/springframework/boot
-:gradle-user-guide: https://docs.gradle.org/4.0.2/userguide
+:gradle-user-guide: https://docs.gradle.org/current/userguide
```

Comment:
This shouldn’t link to current as we’d end up linking to a version we don’t support. If we want to change it, it should point to the latest, supported version. That’s 4.2 at the moment and soon to be 4.3.

Reply:
@wilkinsona Thanks! Changed it to "4.2.1".